src/uct/api/uct.h

```diff
@@ -667,6 +698,70 @@ struct uct_iface_params {
 };
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Parameters for creating a UCT endpoint by @ref uct_ep_create
+ */
+struct uct_ep_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_ep_params_field. Fields not specified by this mask will be
+     * ignored.
+     */
+    uint64_t                          field_mask;
+
+    /**
+     * Interface to create the endpoint on. This is a mandatory field.
+     */
+    uct_iface_h                       iface;
+
+    /**
+     * User data associated with the endpoint.
+     */
+    void                              *user_data;
+
+    /**
+     * The device address to connect to on the remote peer. Should be defined
+     * together with @ref uct_ep_params_t::iface_addr to create an endpoint
+     * connected to a remote interface.
+     */
+    const uct_device_addr_t           *dev_addr;
+
+    /**
+      * Create an endpoint which is connected to remote interface.
+      * The interface address to connect to on the remote peer.
+      * @note requires @ref UCT_IFACE_FLAG_CONNECT_TO_IFACE capability.
+      */
```

Comment:
Suggested re-wording:
This specifies the remote address to use when creating an endpoint that is connected to a remote interface.
`@note`  This requires `@ref` UCT_IFACE_FLAG_CONNECT_TO_IFACE capability.